lib/src/generator/html_generator.dart

```diff
@@ -40,6 +40,8 @@ class HtmlGeneratorBackend extends GeneratorBackendBase {
     var data = PackageTemplateData(options, graph, package);
```

Comment:
If we make `navLinks` a mutable field on PackageTemplateData, then we can edit it here to be `[graph.defaultPackage]` instead of being empty.''

If there are static analysis problems with turning that into a field, you could (😞 ) subclass PackageTemplateData with something like `PackageTemplateDataForSearch extends PackageTemplateData {}` in template_data.dart, and implement `get navLinks`.